project/Dependencies.scala

```diff
@@ -5,34 +5,35 @@ object Dependencies extends DependencyUtils {
   private object Version {
     val akka = "2.6.20"
     val akkaHttp = "10.2.10"
-    val logback = "1.4.4"
-    val sttp = "3.8.3"
-    val http4s = "0.23.16"
-    val enumeratum = "1.7.0"
+    val logback = "1.4.7"
```

Comment:
cant this be 1.4.14?  this project cross compiling?

Reply:
Yes it can, this was just what was recommended as latest